Packages/com.unity.inputsystem/InputSystem/Plugins/iOS/iOSStepCounter.mm

```diff
@@ -0,0 +1,152 @@
+#include <CoreMotion/CoreMotion.h>
+
+#define ENABLE_STEP_COUNTER_LOGGING 1
+#if ENABLE_STEP_COUNTER_LOGGING
+#define STEP_COUNTER_LOG(...) NSLog(@"StepCounter - %@", [NSString stringWithFormat: __VA_ARGS__])
```

Comment:
I don't see any code disabling that, meaning we spam logs in release builds?

Reply:
It's only logging in case of errors

Reply:
Ah ok, missed that